### DISCUSSION TITLE
Staking bug via QLC tracker

### DISCUSSION THREAD

**Discussion by Gyruman** [qlcchain/go-qlc#1395] [General]
Hi,

Recently I tried to invoke staking via the QLC tracker web application.

The staking failed with this message:
![image](https://user-images.githubusercontent.com/5814877/119713041-9375ad00-be61-11eb-98aa-3f0070ba3215.png)

Even though the error, my funds have been drawn from the NEO wallet:
https://neoscan.io/transaction/b781aaeb789c0d3364be69ba5830d8376964d7cf0a4513e39b74e74eb9d3dd2a

Now the funds are at this NEO address:
https://neoscan.io/address/AUVVjKDTpnSPZtXRe1fxUG1N1yCGHVUHt6

When looking at the staking page I see the stake as follows:
![image](https://user-images.githubusercontent.com/5814877/119713296-d3d52b00-be61-11eb-9358-0d56852c24a0.png)

When pressing the button 'continue invoke' I get this error:
![image](https://user-images.githubusercontent.com/5814877/119713407-f49d8080-be61-11eb-9b63-8241b3e0a28c.png)

Now on the recover page I could successfully recover the transaction. Now the staking page shows the staking twice:
![image](https://user-images.githubusercontent.com/5814877/119714841-9d001480-be63-11eb-9c57-f6c1bafb0a7c.png)

Still the balance on the NEO wallet is 249K, which is the correct amount. So no double spending problem.
Wanted to inform you about this issue.

**Comment by zengchen221** [comment]
thank you for your feedback.  there are two transactions on your staking page.  but https://neotube.io/transaction/0x5b0ecaed6ea7d5c192655053cebb85622d32fffdcbd1844d3daef87cc9472605  this transaction failed to execute on neo chain.  so only one transactions is valid
